README.md

```diff
@@ -29,13 +29,13 @@ policies are enforced and propagated as data move from enclave to enclave.
 
 Further information is included in the following documents:
 
+- [Oak Development](docs/development.md) covers practical steps for getting a
+  development Oak system up and running.
 - [Oak Concepts](docs/concepts.md) describes the key concepts involved in Oak
   applications.
 - [Oak ABI](docs/abi.md) documents the core Oak ABI.
 - [Oak SDK](docs/sdk.md) describes the SDK that is provided on top of the Oak
   ABI, to allow more convenient development of Oak applications.
-- [Oak Development](docs/development.md) covers practical steps for getting a
-  development Oak system up and running.
 - [Programming Oak](docs/programming-oak.md) discusses programming for the Oak
   system.
```

Comment:
How come this is moved up? I personally think it's already pretty visible, even at the lower end of the list. — And arguably one would need to understand the Oak concept before starting with Oak development. :]

Reply:
I moved it up to try to compensate for removing the top-level file named `INSTALL.md`; so at least this way the first link that a reader encounters is to the build instructions.  Maybe it needs a reword and/or a rename of `docs/development.md` as well/instead?

Reply:
I see, that's fair. :) I'm fine with having it on top, no string feelings either way.